src/gui/tray/Window.qml

```diff
@@ -21,6 +21,9 @@ Window {
 
     readonly property int maxMenuHeight: Style.trayWindowHeight - Style.trayWindowHeaderHeight - 2 * Style.trayWindowBorderWidth
 
+    Accessible.role: Accessible.Application
+    Accessible.name: qsTr("Nextcloud client tray window")

```

Comment:
I’d say "Nextcloud desktop main dialog" as we call it "main dialog" in the right-click menu?